kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/network/Tab8021xUi.java

```diff
@@ -367,6 +440,18 @@ public void setNetInterface(GwtNetInterfaceConfig config) {
 
         this.username.setValue(config.get8021xConfig().getUsername());
         this.password.setValue(config.get8021xConfig().getPassword());
+
+        if (this.activeConfig.getPassword().isEmpty()
+                || GwtSafeHtmlUtils.htmlUnescape(PASSWORD_PLACEHOLDER).isEmpty()) {
+            this.password.setValue(GwtSafeHtmlUtils.htmlUnescape(PASSWORD_PLACEHOLDER));
+            no8021xPasswordInSnapshot = true;
+        } else {
+            this.password.setValue(GwtSafeHtmlUtils.htmlUnescape(this.activeConfig.getPassword()));
```

Comment:
Do we want to replace `&lt;` with `<` and so on in passwords? If I insert `mypassword&lt;` then it will become `mypassword<`